mappings/net/minecraft/client/option/GameOptions.mapping

```diff
@@ -2,39 +2,69 @@ CLASS net/minecraft/class_315 net/minecraft/client/option/GameOptions
 	FIELD field_1814 cloudRenderMode Lnet/minecraft/class_4063;
 	FIELD field_1815 hideServerAddress Z
 	FIELD field_1816 keySmoothCamera Lnet/minecraft/class_304;
+		COMMENT A key binding for toggling smooth camera.
+		COMMENT Not bound to any keys by default.
 	FIELD field_1817 chatLinksPrompt Z
 	FIELD field_1818 showSubtitles Z
 	FIELD field_1819 forceUnicodeFont Z
 	FIELD field_1820 chatOpacity D
 	FIELD field_1822 keyInventory Lnet/minecraft/class_304;
+		COMMENT A key binding for opening {@linkplain net.minecraft.client.gui.screen.ingame.InventoryScreen the inventory screen}.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_E the E key} by default.
 	FIELD field_1823 GSON Lcom/google/gson/Gson;
 	FIELD field_1824 keyTogglePerspective Lnet/minecraft/class_304;
+		COMMENT A key binding for toggling perspective.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_F5 the F5 key} by default.
 	FIELD field_1825 chatHeightUnfocused D
 	FIELD field_1826 fov D
 	FIELD field_1827 advancedItemTooltips Z
 	FIELD field_1828 fullscreenResolution Ljava/lang/String;
 	FIELD field_1829 mainArm Lnet/minecraft/class_1306;
 	FIELD field_1830 realmsNotifications Z
 	FIELD field_1831 keySwapHands Lnet/minecraft/class_304;
+		COMMENT A key binding for swapping the items in the main hand and the off hand.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_F the F key} by default.
```

Comment:
```suggestion
		COMMENT A key binding for swapping the items in the selected slot and the off hand.
		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_F the F key} by default.
		COMMENT
		COMMENT <p>The selected slot is the slot the mouse is over when in a screen.
		COMMENT Otherwise, it is the main hand.
```
See https://github.com/FabricMC/yarn/pull/2640#discussion_r688859220.